suite-native/fiat-rates/src/fiatRatesSelectors.ts

```diff
@@ -9,7 +9,15 @@ import {
     selectAccounts,
     selectTransactions,
 } from '@suite-common/wallet-core';
-import { Account, AccountKey, WalletAccountTransaction } from '@suite-common/wallet-types';
+import {
+    Account,
+    AccountKey,
+    TokenAddress,
+    TokenSymbol,
+    WalletAccountTransaction,
+} from '@suite-common/wallet-types';
+import { isEthereumAccountSymbol } from '@suite-native/ethereum-tokens';
```

Comment:
no imports from suite-native here, we will move this package to common soon, so please move all this selectors to native/eth-tokens package

Reply:
[f1d4ecd](https://github.com/trezor/trezor-suite/pull/8142/commits/f1d4ecd63e11a5c9fd817bcf0ad04b98f683d5e2)